Arguments: 
  /usr/bin/node /usr/bin/yarn add @auth0/auth0-js

PATH: 
  /home/grzybek/.cargo/bin:/home/grzybek/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/home/grzybek/.dotnet/tools:/opt/flutter/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/var/lib/snapd/snap/bin:/home/grzybek/.local/share/JetBrains/Toolbox/scripts

Yarn version: 
  1.22.19

Node version: 
  19.8.1

Platform: 
  linux x64

Trace: 
  Error: https://registry.yarnpkg.com/@auth0%2fauth0-js: Not found
      at params.callback [as _callback] (/usr/lib/node_modules/yarn/lib/cli.js:66145:18)
      at self.callback (/usr/lib/node_modules/yarn/lib/cli.js:140890:22)
      at Request.emit (node:events:512:28)
      at Request.<anonymous> (/usr/lib/node_modules/yarn/lib/cli.js:141862:10)
      at Request.emit (node:events:512:28)
      at IncomingMessage.<anonymous> (/usr/lib/node_modules/yarn/lib/cli.js:141784:12)
      at Object.onceWrapper (node:events:626:28)
      at IncomingMessage.emit (node:events:524:35)
      at endReadableNT (node:internal/streams/readable:1359:12)
      at process.processTicksAndRejections (node:internal/process/task_queues:82:21)

npm manifest: 
  {
  	"name": "frontend",
  	"version": "0.0.1",
  	"private": true,
  	"scripts": {
  		"dev": "vite dev",
  		"build": "vite build",
  		"preview": "vite preview"
  	},
  	"devDependencies": {
  		"@sveltejs/adapter-auto": "^2.0.0",
  		"@sveltejs/kit": "^1.5.0",
  		"svelte": "^3.54.0",
  		"vite": "^4.2.0"
  	},
  	"type": "module",
  	"dependencies": {
  		"@auth0/auth0-spa-js": "^2.0.4",
  		"auth0-js": "^9.20.2"
  	}
  }

yarn manifest: 
  No manifest

Lockfile: 
  No lockfile
